lxc/copy.go

```diff
@@ -272,21 +300,49 @@ func (c *cmdCopy) copyInstance(conf *config.Config, sourceResource string, destR
 			entry.Profiles = []string{}
 		}
 
+		// If there are device overrides that are expected to be applied to profile devices then load the profiles
```

Comment:
Can we avoid this duplication by making a temporary function that can be called from both places?